xl/formatter.py

```diff
@@ -410,7 +410,8 @@ def format(self, track, markup_escape=False):
 
         return Formatter.format(self)
 
-class TagFormatter():
+# XXX: should inherite from class Formatter?
```

Comment:
Well, the thing is since the other classes in the module all are subclasses of Formatter base class, I don't know if it's a mistake or deliberate. Only if it's deliberate, it should inherit from object.